launcher/mainwindow_moc.cpp

```diff
@@ -116,10 +116,43 @@ MainWindow::MainWindow(QWidget * parent)
 
 	ui->settingsView->setDisplayList();
 	
+	if(settings["launcher"]["preLauncher"].Bool()) // MessageBox before launcher asking how to start (automatically starts client after 10 seconds)
+		preLauncher();
+
 	if(settings["launcher"]["updateOnStartup"].Bool())
 		UpdateDialog::showUpdateDialog(false);
 }
 
+void MainWindow::preLauncher()
+{
+	QMessageBox msgBox;
+	int secondsRemaining = 10;
+	QString text = tr("How to start?\n\nVCMI starts automatically in ");
+	QString textSeconds = tr(" Seconds");
+	msgBox.setText(text + QString::number(secondsRemaining) + textSeconds);
```

Comment:
```suggestion
    QString textTemplate = tr("How to start?\n\nVCMI starts automatically in %1 seconds");
	msgBox.setText(textTemplate.arg(secondsRemaining));
```

Can be improved further - to allow plural translations (1 second, 2 seconds) but not sure about Qr syntax for that